### PR TITLE
Compatibility with Asciidoctor.js 1.5.7

### PR DESCRIPTION
**Pull request by Mogztter**

### CODE REVIEW

File: test/shared.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ alice -> bob
 @enduml`
 sharedSpec.LOCAL_URL = 'http://localhost:8080'
 sharedSpec.PLANTUML_REMOTE_URL = 'http://www.plantuml.com/plantuml'
-sharedSpec.PNG_DIAGRAM_SIZE = 1787
+sharedSpec.PNG_DIAGRAM_SIZE = 1790
```
Comment (Mogztter) [inline test/shared.js]: Not sure about this one but on my machine, the diagram size is 1790 :thinking:

**Comment by mojavelinux** [comment]
Do you think it's necessary to support both 1.5.6 and 1.5.7? For example, Antora is currently locked to 1.5.6, so I think there will be at least be a short transition period when this is necessary.

**Comment by Mogztter** [comment]
Yes, it should be a major version.
Or we could convert the hash to an object in order to handle both versions.

**pseudo-code**
```js
var fromHash = function (hash) {
  var object = {};
  var data = hash.$$smap;
  for (var key in data) {
    object[key] = data[key];
  }
  return object;
};

if (typeof attrs === 'object' && '$$smap' in attrs) {
  attrs = fromHash(attrs)
}
```

a check on the attributes object. If the objects is an `Opal.hash` then we should use `Opal.hash_get` and

**Comment by mojavelinux** [comment]
> Or we could convert the hash to an object in order to handle both versions.

That's what I think we should do for now.

**Comment by eshepelyuk** [comment]
Looks fine, since @mojavelinux @Mogztter are able to merge - go ahead. Let me know when I can release a new version.

**Comment by Mogztter** [comment]
Done, the code is now compatible with both Asciidoctor.js 1.5.6 and Asciidoctor.js 1.5.7.

**Comment by Mogztter** [comment]
@eshepelyuk @mojavelinux can I proceed and merge this pull request ?

**Comment by mojavelinux** [comment]
It's up to @eshepelyuk. I give it a :+1:.

**Comment by eshepelyuk** [comment]
@Mogztter  Please proceed. I'll try to release a new version in couple of days (on a business trip now, so may be delayed)

**Comment by Mogztter** [comment]
Excellent thanks @eshepelyuk

**Pull request merged**